backend/compact-connect/lambdas/provider-data-v1/data_model/client.py

```diff
@@ -95,19 +96,36 @@ def get_providers_sorted_by_family_name(
             self, *,
             compact: str,
             dynamo_pagination: dict,
+            provider_name: tuple[str, str] = None,  # (givenName, familyName)
             jurisdiction: str = None,
             scan_forward: bool = True
     ):  # pylint: disable-redefined-outer-name
         logger.info('Getting providers by family name')
+
+        # Create a name value to use in key condition if name fields are provided
+        name_value = None
+        if provider_name is not None and provider_name[0] is not None:
+            name_value = f'{quote(provider_name[0])}#'
+            # We won't consider givenName if familyName is not provided
+            if provider_name[1] is not None:
+                name_value += f'{quote(provider_name[1])}#'
```

Comment:
Isn't this doing the opposite of the comment? Only considering familyName if givenName is provided?

Reply:
Ah I think the comment on line 99 is reversed?

Reply:
Oh, sure is. Good catch.